### PR TITLE
feat: ensure liquidity mining NFT id is within reserved range

### PR DESCRIPTION
**Pull request by Roznovjak**
Fixes: #427

### CODE REVIEW

File: pallets/liquidity-mining/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-liquidity-mining"
-version = "1.0.1"
+version = "1.0.2"
```
Comment (lumir-mrkva) [inline pallets/liquidity-mining/Cargo.toml:3]: as this change adds another feature, i would increase version to `1.1.0`

**Comment by lumir-mrkva** [comment]
@Roznovjak can you add test that check works like expected?

**Comment by Roznovjak** [comment]
> @Roznovjak can you add test that check works like expected?

A test is automatically generated from it. But it's more like a verification, because testing it (both scenarios) would require two runtimes with different pallet configurations.

**Comment by lumir-mrkva** [comment]
> > @Roznovjak can you add test that check works like expected?
> 
> A test is automatically generated from it. But it's more like a verification, because testing it (both scenarios) would require two runtimes with different pallet configurations.

when it actually panics? after runtime upgrade?

**Comment by Roznovjak** [comment]
> > > @Roznovjak can you add test that check works like expected?
> > 
> > 
> > A test is automatically generated from it. But it's more like a verification, because testing it (both scenarios) would require two runtimes with different pallet configurations.
> 
> when it actually panics? after runtime upgrade?

It looks like it behaves just like a normal test. [Ref](https://github.com/paritytech/substrate/pull/6356)

**Review by lumir-mrkva — APPROVED** [review]
found the test: `test mock::__construct_runtime_integrity_test::runtime_integrity_tests ... ok`

LGTM

**Pull request merged**